c/frame/rbind.cc

```diff
@@ -295,6 +295,7 @@ void DataTable::rbind(
   }
   ncols_ = new_ncols;
   nrows_ = new_nrows;
+  nkeys_ = 0; // this will drop all the keys, if there were any
```

Comment:
We should prefer to throw an error when a user tries to rbind a keyed frame, similarly to how we do not allow resizing a keyed frame:
```
>>> import datatable as dt
>>> DT = dt.Frame(A=range(5))
>>> DT.key = 'A'
>>> DT.nrows = 10
ValueError: Cannot increase the number of rows in a keyed frame
```
However, this can be deferred to a later PR

Reply:
Yes, this is a temporary fix. I will create an issue and list cases where operations on the keyed frame should be adjusted.